htlcswitch/link.go

```diff
@@ -545,7 +545,8 @@ func (l *channelLink) WaitForShutdown() {
 func (l *channelLink) EligibleToForward() bool {
 	return l.channel.RemoteNextRevocation() != nil &&
 		l.ShortChanID() != hop.Source &&
-		l.isReestablished()
+		l.isReestablished() &&
+		!l.IsFlushing(Outgoing)
```

Comment:
This will prevent the link from sending `update_fee` after we've sent `shutdown` since `updateChannelFee` calls `EligibleToForward`.  Sending `update_fee` during the coop close process was brought up in prior spec meetings since you may want to adjust the commitment feerate in case you need to force close

Reply:
Seems like `update_fee` shouldn't be gated on "EligibleToForward"... I'll take a look at how we use it in that way and develop a different measurement for update_fee